widgets/common/src/blueprintMarketplace/blueprintMarketplaceTabsConfig.ts

```diff
@@ -1,7 +1,7 @@
 import { times } from 'lodash';
 
 const t = Stage.Utils.getT(`widgets.blueprints.configuration.marketplaceTabs`);
-const defaultNumberOfTabs = 6;
+const defaultNumberOfTabs = 13;
```

Comment:
There is [a neat option](https://www.i18next.com/translation-function/objects-and-arrays) to extract an `object`/`array` from the `i18n` 

We could try to use it here, just so we wouldn't need to remember about adjusting this number in the future 💪 

```suggestion
import { times, size } from 'lodash';

const t = Stage.Utils.getT(`widgets.blueprints.configuration.marketplaceTabs`);
const defaultNumberOfTabs = size(t('columns.defaultValues', { returnObjects: true }));
```

Reply:
done 👌